tests/hierarchy_driver.cc

```diff
@@ -212,7 +228,9 @@ int main(int argc, char *argv[])
   auto const params = std::make_shared<boost::property_tree::ptree>();
   boost::property_tree::info_parser::read_info(filename, *params);
 
-  int const fe_degree = params->get<unsigned int>("laplace.fe_degree", 1);
+  int const fe_degree = params->get<unsigned int>("laplace.fe_degree", 4);
+  params->put("fast_ap", true);
+  params->put("eigensolver.type", "lanczos");
```

Comment:
What was the default?

Reply:
`lanczos` for matrix-free and `arpack` for matrix-based. I just like to be explicit about the parameters we are using here.